dlh_utils/tests/test_profiling.py

```diff
@@ -0,0 +1,172 @@
+'''
+Pytesting on Profiling functions
+'''
+
+import pyspark
+from pyspark.sql import SparkSession
+import pyspark.sql.functions as F
+from pyspark.sql.functions import col
```

Comment:
Functions are imported in line 7, so this repeats the import. Can you replace use of "col" with "F.col "?

Reply:
changed to import pyspark.sql.functions as F, it didnt like F.col